cdap-api/src/main/java/co/cask/cdap/api/metadata/Metadata.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright © 2016 Cask Data, Inc.
+ * Copyright © 2016-2018 Cask Data, Inc.
```

Comment:
unnecessary change?

Reply:
There were couple of files changed in 2018 without license year changes.